extensions/spark/install_spark.sh

```diff
@@ -128,7 +128,6 @@ export SPARK_DAEMON_MEMORY=${SPARK_DAEMON_MEMORY}m
 export SPARK_WORKER_DIR=${SPARK_WORKDIR}
 export SPARK_LOCAL_DIRS=${SPARK_TMPDIR}
 export SPARK_LOG_DIR=${SPARK_LOG_DIR}
-export SPARK_CLASSPATH=\$SPARK_CLASSPATH:${LOCAL_GCS_JAR}
```

Comment:
I deleted this line because SPARK_CLASSPATH was deprecated in Spark 1.0+. When I set the SPARK_CLASSPATH variable, Spark wasn't picking it up. 

http://apache-spark-user-list.1001560.n3.nabble.com/SPARK-CLASSPATH-Warning-td9233.html